### ISSUE TITLE
Testing ZIP installs in CI

### ISSUE THREAD

**Issue by MTCam** [illinois-ceesd/mirgecom#235]
Thermochem config file management (not yet merged) is *supposed* to work even if using a ZIP-installed _MIRGE-Com_.  Is there any way to test that in CI? Any ideas, @matthiasdiener?

**Comment by matthiasdiener** [comment]
Emirge's CI contains a test for the modules.zip that it creates during installation (see https://github.com/illinois-ceesd/emirge/blob/master/.github/workflows/makezip.yaml). However, modules.zip does not contain mirgecom itself at the moment, just its dependencies.

**Comment by MTCam** [comment]
Yeah thanks. It looks like I could likely harness that (with customization of `makezip.sh`) to add ZIP install testing in CI for _MIRGE-Com_, however I am loathe to add yet another potentially lengthy CI construct for a simple test.  

Would it make any sense at all to add a _MIRGE-Com_ ZIP install test to `emirge` CI?
